### ISSUE TITLE
License policy

### ISSUE THREAD

**Issue by skreutzer** [luarocks/luarocks-site#123]
Could you please point out somewhere, at least on the About page, which licenses are permitted, if the operation has a license policy at all? With package managers of GNU/Linux distros, one can be/make sure that only 100% free/libre licensed software is installed from the repositories, but I'm really suspicious regarding `luarocks`.

**Comment by leafo** [comment]
We don't have any specific license requirements as far as I know. The license is set on the rockspec of the module: https://github.com/luarocks/luarocks/wiki/Rockspec-format

**Comment by hishamhm** [comment]
Yes, the license is up to the author of each rockspec with no specific requirements, as much as it is up to the author of each Github repo on this very site we're all in. :)
